datalad_registry_client/opts.py

```diff
@@ -64,17 +66,23 @@ def process_args(
         if sibling not in remotes:
             raise ValueError("Unknown sibling: {}".format(sibling))
 
-    if not url:
-        if not sibling:
-            raise ValueError(
-                "Must specify URL to use when sibling isn't given")
+    urls: List[Tuple[str, str]] = []
+    if url:
+        urls.append((url, url_encode(url)))
```

Comment:
Why bother `url_encode`ing in this function (in 3 places) - why not to return just a list of `urls` and invoke `url_encode` in one spot whenever used outside and encoded value is needed?

Reply:
Done.